### DISCUSSION TITLE
Arranging items in the design view

### DISCUSSION THREAD

**Discussion by Mastomaki** [Spine-project/Spine-Toolbox#2545] [Support]
Is there a way to conveniently arrange items in the design view? E.g. move several items at the same time?

**Comment by manuelma** [comment]
You can "rubber band" select (Ctrl + drag the mouse), or select the items one by one by clicking on them while holding the Ctrl key. Then drag any of the selected items.

**Reply by Mastomaki** [reply]
Thanks! Works very well.
